aptible/client_utils_test.go

```diff
@@ -100,8 +100,12 @@ func TestGenerateErrorFromClientError(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			err := generateErrorFromClientError(tt.args.abstractedError)
-			wantErr := err == nil
-			if wantErr == tt.wantErr && tt.errorBody != err.Error() {
+			gotErr := err != nil
+			if tt.wantErr != gotErr {
+				t.Errorf("wanted an error (tt.wantErr), but did not get an error (gotErr) OR didn't want an error" +
+					"and got an error!")
+			}
+			if tt.wantErr == gotErr && tt.errorBody != err.Error() {
```

Comment:
Oh `tt.wantErr == gotErr` is implied in the previous `if`

Reply:
🤔 we want this to guard against `err.Error()` if we do not want an error or the test suite will panic

Reply:
I see what you're saying then it should be 

```suggestion
			if gotErr && tt.errorBody != err.Error() {
```

Reply:
I'm okay with that. I think the preceding if statement would act a guard for this and errors appropriately there so I'm gonna commit this.